rudder-core/src/main/scala/com/normation/rudder/services/policies/InterpolatedValueCompiler.scala

```diff
@@ -253,31 +310,29 @@ class InterpolatedValueCompilerImpl extends RegexParsers with InterpolatedValueC
   //exactly quote s - no regex authorized in
   def id(s: String) = ("""(?iu)\Q""" + s + """\E""").r
 
-  def all: Parser[List[Token]] = (rep1(interpol | plainString) | emptyVar)
+  def all: Parser[List[Token]] = (rep1(interpol | nodeProp | plainString) | emptyVar)
 
   //empty string is a special case that must be look appart from plain string.
   //also parse full blank string, because no need to look for more case for them.
-  def emptyVar: Parser[List[CharSeq]] = """(?iums)(\s)*""".r ^^ { x => List(CharSeq(x)) }
+  def emptyVar   : Parser[List[CharSeq]] = """(?iums)(\s)*""".r ^^ { x => List(CharSeq(x)) }
 
-  def plainString: Parser[CharSeq] = ("""(?iums)((?!\Q${rudder.\E).)+""").r  ^^ { CharSeq(_) }
+  // plain string must not match our identifier, ${rudder.* and ${node.properties.*}
+  def plainString: Parser[CharSeq] = ("""(?iums)((?!(\Q${rudder.\E|\Q${node.properties\E)).)+""").r  ^^ { CharSeq(_) }
 
   //identifier for step in the path or param names
-  def propId: Parser[String] = """[\-_a-zA-Z0-9]+""".r
+  def propId     : Parser[String] = """[\-_a-zA-Z0-9]+""".r
```

Comment:
Internet seems to say that multiline keys are invalid in json : http://stackoverflow.com/questions/2392766/multiline-strings-in-json , so indeed there is no need for multiline for string, but your proposition around the [ ] will help readability
I have a question for the non greedy, i'm not sure of the behaviour of the parser, but if it is not greedy, isn't there a risk to match partial string ? (it seems not, but a test with keys data and datacenter would ensure that future refactoring don't break it)